pkg/bootstrap/docker/up.go

```diff
@@ -917,7 +919,18 @@ func (c *ClientStartConfig) ImportTemplates(out io.Writer) error {
 	if err := c.importObjects(out, openshiftNamespace, templateLocations); err != nil {
 		return err
 	}
-	return c.importObjects(out, "kube-system", adminTemplateLocations)
+	version, err := c.OpenShiftHelper().ServerVersion()
+	if err != nil {
+		return err
+	}
+	if shouldImportAdminTemplates(version) {
+		return c.importObjects(out, "kube-system", adminTemplateLocations)
+	}
+	return nil
+}
+
+func shouldImportAdminTemplates(v semver.Version) bool {
+	return v.GT(openshiftVersion35)
```

Comment:
you're right ... no, it needs to be 3.6 ... will change the condition to >= 3.6.0